### ISSUE TITLE
F3X cloning broke for some games

### ISSUE THREAD

**Issue by malakeifox2** [F3XTeam/RBX-Building-Tools#142]
in some games made by my friends, my f3x when cloning a part suddenly unselects it. same with creating parts, it also seemed to work fine on parts already in game, any ideas how could this be fixed?

**Comment by asd2222222q** [comment]
have you found the solution?

**Comment by malakeifox2** [comment]
> have you found the solution?

no, but i did find out that on older games, the bug is not present. so i just make games using the old games which i strip down to baseplate and build off of
